spin/apps/02b-json-api-openai/spin.toml

```
@@ -0,0 +1,23 @@
+spin_manifest_version = "1"
+authors = ["teknicus"]
+description = ""
+name = "magic-8-ball"
+trigger = { type = "http", base = "/" }
+version = "0.1.0"
+
+[[component]]
+id = "magic-8-ball"
+source = "target/magic-8-ball.wasm"
+exclude_files = ["**/node_modules"]
+allowed_http_hosts = ["api.openai.com"]
+[component.trigger]
+route = "/magic-8"
+[component.build]
+command = "npm run build"
```

add API key via config
```suggestion
command = "npm run build"
[component.config]
openai_key = "{{ openai_key }}"
```